stmt/stmts/insert.go

```diff
@@ -46,15 +46,20 @@ const (
 	DelayedPriority
 )
 
+// InsertRest is the rest part of insert/replace into statement.
+type InsertRest struct {
```

Comment:
The InsertRset is very close to Insert and replace and can't be moved to rset package, maybe, so let it here now. 
btw, InsertRest -> InsertRset @zimulala